integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala

```diff
@@ -806,8 +788,19 @@ object BloopDefaults {
         val isForkedExecution = (Keys.fork in configuration in forkScopedTask).value
         val workingDir = if (isForkedExecution) Keys.baseDirectory.value else rootBaseDirectory
         val extraJavaOptions = List(s"-Duser.dir=${workingDir.getAbsolutePath}")
+        lazy val runtimeClasspath = Keys.fullClasspath.in(Runtime).value.map(_.data.toPath()).toList
+        lazy val javaRuntimeHome = (Keys.javaHome.in(Runtime) ).value.map(_.toPath())
+        lazy val javaRuntimeOptions = (Keys.javaOptions.in(Runtime)).value
```

Comment:
The `in` syntax is deprecated since sbt `1.5.0`. Can we use the slash syntax instead.

```suggestion
        lazy val runtimeClasspath = (Runtime /  Keys.fullClasspath).value.map(_.data.toPath()).toList
        lazy val javaRuntimeHome = (Runtime / Keys.javaHome).value.map(_.toPath())
        lazy val javaRuntimeOptions = (Runtime / Keys.javaOptions).value
```

Reply:
Thanks! Fixed here and in any code I added